.gitignore

```diff
@@ -9,6 +9,9 @@
 !*.md
 !*.rst
 
+# Don't ignore makefiles
+!Makefile
+
```

Comment:
Should we be adding statements to here for also not ignoring the new docker and CI files?

Reply:
At some point possibly